code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -3,25 +3,13 @@
 	density = 1
 	anchored = 1
 	icon = 'icons/obj/cult.dmi'
-	var/cooldowntime = 0
-	var/health = 100
-	var/death_message = "<span class='warning'>The structure falls apart.</span>" //The message shown when the structure is destroyed
-	var/death_sound = 'sound/items/bikehorn.ogg'
```

Comment:
not before. I pretty much ave this in two types prop type used on maps for show (or if an admin is filling funny) and the one for cult game moder proper. Why? I hate map conflicts....

also health may be buggy, TG does have a system for it but it might be too much to do in this PR for right now.